tox.ini

```diff
@@ -0,0 +1,13 @@
+[tox]
+envlist = py310
+
+[testenv]
+deps =
+	pytest
+commands =
+    pytest
+
+[pytest]
+minversion = 7.2.2
+testpaths =
+    tests
```

Comment:
This can happen in a separate PR, if we want to land this one first, but since we have a couple tests, it would be nice to run these with GH actions.